mobile_files/yarn.lock

```diff
@@ -59,12 +59,12 @@
     source-map "^0.5.0"
     trim-right "^1.0.1"
 
-"@babel/generator@^7.1.6":
```

Comment:
Curious as to why this particular PR requires updating `yarn.lock`. If not strictly related, should it be done in a separate PR?

Reply:
That's because I was having yarn throw errors at me about unpacking, and the suggestion was to try to refresh `yarn.lock`, but I can try rebasing with latest `develop` and see if that issue is gone.